Godeps/Godeps.json

```
@@ -45,8 +46,8 @@
 		},
 		{
 			"ImportPath": "github.com/satori/go.uuid",
-			"Comment": "v1.1.0",
-			"Rev": "879c5887cd475cd7864858769793b2ceb0d44feb"
+			"Comment": "v1.2.0-12-gb2ce238",
```

Same question here; would the last tagged version of this work instead?

```suggestion
			"Comment": "v1.2.0",
```

https://github.com/satori/go.uuid/compare/v1.2.0...master

Perhaps we should replace `github.com/satori/go.uuid` with `github.com/google/uuid`, which is commonly used in other projects (e.g. in containerd), and looks to be more actively maintained; https://github.com/google/uuid